docker/rails/TestSite/public/index.html

```diff
@@ -0,0 +1,9 @@
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>test</title>
+</head>
+<body>
+<h1>Hello WOVN.php world!</h1>
```

Comment:
It is better to change from `WOVN.php` to `wovnrb` or remove `WOVN.php`.

Reply:
Yes I agree it feels weird. I left it that way because I was using PHP as the reference implementation and I want to keep everything the same. But now that PHP version also needs update we can just have a new page to do that.